### PR TITLE
5_9_1 and 5_9_2

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_9_1.txt [hunk 1]
```diff
@@ -247,12 +257,13 @@ Three seconds later.
 [sha:a7cf29f7217ee3899225d7d0581a6d7d53beb1d4]{
 -- Page 790, Offset 10799.
 -- 空気抵抗による失速を力でねじ伏せるように、一投目はヴローヴの頭部に飛来し、
-As though overwhelming the attempts of air resistance to slow it down through sheer power, the first throw hurtles toward Vlov's head,
+Surpassing air resistance as though forcing it into submission through sheer power,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_1.txt:260]: I think I prefer orig here

**Review by rschlaikjer — APPROVED** [review]
otsucurry

File: script/Arcueid/Day 5/05_00_ARC05_9_1.txt [hunk 2]
```diff
@@ -136,12 +139,12 @@ So if possible, she should throw three pieces of debris.
...
 -- Page 781, Offset 10777.
 -- 一発目はヴローヴを直接狙い、
-The first shot should be aimed directly at Vlov,
+The first shot should be aimed directly at Vlov―――
 }
 [sha:84f93710c2edd6d0fa271e2dd3e920b48d04b600]{
 -- Page 781, Offset 10778.
 -- 二発目はわざと狙いを外してヴローヴの後方に、
-the second shot should deliberately miss and land behind him,
+the second shot should deliberately miss and land behind him―――
```
Comment (nolo16) [inline script/Arcueid/Day 5/05_00_ARC05_9_1.txt:147]: should this be capitalized? ross said he's been capitalizing lines like these but i'm not sure if this qualifies

File: script/Arcueid/Day 5/05_00_ARC05_9_1.txt [hunk 3]
```diff
@@ -347,12 +359,12 @@ It is impossible for a human to cross those two steps.
...
 -- Page 797, Offset 10819.
 -- 悲鳴をあげるだけの余裕も、
-My ability to scream,
+My ability to scream―――
 }
 [sha:1a83fcc05740cf8545d9966a645e2c3ef3ed2347]{
 -- Page 797, Offset 10820.
 -- 苦痛を感じるだけの感覚も、既に凍っていた。
-and my capacity to feel pain were already frozen solid.
+and my capacity to feel pain are already frozen solid.
```
Comment (nolo16) [inline script/Arcueid/Day 5/05_00_ARC05_9_1.txt:367]: caps

**Review by nolo16 — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_9_1.txt (hunk 2 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9_1.txt:147]: Nope.

File: script/Arcueid/Day 5/05_00_ARC05_9_1.txt (hunk 3 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9_1.txt:367]: no need for caps here, its the same sentence; it's just split

Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_9_1.txt:367]: We should probably actually agree on the rules here. I have been operating under "every new line starts with a capital letter"

Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_9_1.txt:367]: I don't really care all that much but it's not really a new sentence or an independent clause so I don't see a reason to capitalize it

**Pull request merged**